packages/css/src/components/link-list/link-list.template.ts

```diff
@@ -23,10 +25,7 @@ export function linkListTemplate({ type, links }: LinkList) {
   const children = html`
     ${links.map(link => html`
       <li>
-        <a
-          href=${link.url}
-          class=${ifDefined(link.modifier)}
-        >${link.label}</a>
+        ${anchorTemplate({ url: link.url, modifier: link.modifier, label: link.label })}
```

Comment:
Volgens mij is `link` van het type `Anchor`, en kan je dus het hele object in een keer doorgeven aan `anchorTemplate()`